src/pyJiraCli/issue_constants.py

```diff
@@ -47,14 +47,6 @@
     '8' : 'QMeldung'
 }
 
-ISSUE_PRIORITIES = {
```

Comment:
Are any other constants required? Can we remove the whole file?

Reply:
The import function still uses the issue constants. This should be addressed in a seperate issue